File: Web Charge/Examples.Charge.Application/Messages/Request/PhoneRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Examples.Charge.Application.Messages.Request
{
    public class PhoneRequest
    {
        public int PersonId { get; set; }
        public int PhoneTypeId { get; set; }
        public string Number { get; set; }
    }
}

File: stefanini-front/src/app/models/requests/phone.request.ts
export class PhoneRequest {
  personId: number;
  phoneTypeId: number;
  number: string;

  constructor(params: Partial<PhoneRequest>) {
    this.personId = +params.personId;
    this.phoneTypeId = +params.phoneTypeId;
    this.number = params.number;
  }
}
